TFGapp/TFGapp/Presentacion.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+#include <string>
+#include <iostream>
+#include <vector>
+#include "Profesor.h"
+
+using namespace std;
+
+class Presentaciom {
+
+private:
+
+	string ID;
+	int hora;
+	int dia;
+	int aula;
+	int slot;
+	int convocatoria;
+	vector<Profesor> tribunal; //Lista de profesores que acuden a la presentaci�n
```

Comment:
```suggestion
	vector<Profesor *> tribunal; //Lista de profesores que acuden a la presentación
```
La lista debería ser de punteros pq no queremos tener varias copias de un mismo profesor